modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/NumberInputTest.ts

```diff
@@ -418,6 +421,19 @@ describe('browser.tinymce.themes.silver.throbber.NumberInputTest', () => {
     TinyAssertions.assertContent(editor, `<p style="font-size: ${originalFontSize};">a<span style="font-size: ${originalFontSize};">b</span>c</p>`);
   });
 
+  it('TINY-10330: the presence and updates of fontsizeinput should not trigger warnings', async () => {
+    chai.use(spies);
+    const spy = chai.spy.on(console, 'warn');
```

Comment:
I wonder if we will be using this enough to warrant a library to have to bump etc since this is the first time we are in the need of this. We could just do a spy thing locally in this test just patch and un-patch the console.warn function?

Reply:
I tried to refactor in this way:
```
// eslint-disable-next-line no-console
    const oldConsoleWarn = console.warn;
    let isWarnCalled = false;
    // eslint-disable-next-line no-console
    console.warn = (a) => {
      isWarnCalled = true;
      oldConsoleWarn(a);
    };

    const editor = hook.editor();
    editor.setContent('<p style="font-size: 20px;">abc</p>');
    TinySelections.setSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 2);
    const input = TinyUiActions.clickOnToolbar<HTMLInputElement>(editor, '.tox-number-input input');
    await Waiter.pTryUntilPredicate(`Wait for the new input value is setted`, () => input.dom.value === '20px');

    expect(isWarnCalled).to.eq(false);
    // eslint-disable-next-line no-console
    console.warn = oldConsoleWarn;
```

and it seems to work, we could change in this way or add the library so we can se it also for the future tests

I this the lib could be useful but I don't have strong opinion on it